admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -124,11 +123,13 @@
   import {
     getProductImages,
     saveImageInformations,
-    replaceImage,
     saveImagePosition,
+    replaceImage,
+    removeProductImage,
   } from '@pages/product/services/images';
   import ProductMap from '@pages/product/product-map';
   import ProductEventMap from '@pages/product/product-event-map';
+  import ConfirmModal from '@components/modal';
```

Comment:
Oh one question I forgot, we have a Vue component for modal here `admin-dev/themes/new-theme/js/vue/components/Modal.vue` Maybe we should use it?

Reply:
Maybe, gonna take a look